api/notd/api/models_v1.py

```diff
@@ -60,3 +60,11 @@ class ApiSponsoredToken(BaseModel):
     collection: ApiCollection
     date: datetime.datetime
     latestTransfer: Optional[ApiTokenTransfer]
+
+class ApiCollectionActivity(BaseModel):
+    date: datetime.date
+    totalVolume: int
+    transferCount: int
```

Comment:
these cant be ints in the API - they are too big for JSON int values. they need to be strings